sdk/core/core-http/Changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.0.0-preview2 - 2019-08-05
+
+- Removed `ServiceClientCredentials` type from `credentials` parameter of `ServiceClient` ([#4367](https://github.com/Azure/azure-sdk-for-js/pull/4367)).  Credential implementations are now standardized on `@azure/core-auth`'s `TokenCredential` interface and provided by `@azure/identity`.
+- Added an `AccessTokenCache` so that access tokens can be cached across pipeline instances ([#4174](https://github.com/Azure/azure-sdk-for-js/pull/4174)).
```

Comment:
```suggestion
- Added an `AccessTokenCache` so that access tokens can be cached across pipeline instances ([PR #4174](https://github.com/Azure/azure-sdk-for-js/pull/4174)).
```